tmdb/src/main/java/me/banes/chris/tivi/tmdb/TmdbImageProviderRepo.kt

```diff
@@ -30,7 +30,7 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class TmdbManager @Inject constructor(
+class TmdbImageProviderRepo @Inject constructor(
```

Comment:
This should stay as `TmdbManager`. While it only currently provides the TmdbImageProvider, it will do more in the future (possible TMDb login, etc).

Reply:
done